docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       - 3000
     command: ./script/run-docker-nightwatch.sh
   selenium-chrome:
-    image: selenium/standalone-chrome
+    image: selenium/standalone-chrome:3.14.0-francium
```

Comment:
How bizarre!

We should probably make a reminder of some kind to loop back around to this at some point and update it so we're not stuck with an old version for too long.